xdggs/accessor.py

```diff
@@ -115,3 +116,40 @@ def cell_boundaries(self):
         return xr.DataArray(
             boundaries, coords={self._name: self.cell_ids}, dims=self.cell_ids.dims
         )
+
+    def explore(
+        self, *, cell_boundaries="cell_boundaries", cmap="viridis", center=None
+    ):
+        """interactively explore the data using `lonboard`
+
+        Requires `geopandas`, `matplotlib`, and `lonboard` to be installed.
```

Comment:
FYI as of the next Lonboard version (hopefully released this week), this will also require `pyarrow` to be explicitly installed. `pyarrow` is a really massive dependency, and so I don't want to force all users to have it installed. But I haven't implemented my own version of converting GeoPandas to Arrow, and so I require pyarrow for that.

Reply:
thanks for the warning, I'll extend this list here, or we can try to work around geopandas (see below)